src/workloads/execution/VariadicAggregateExpressions.yml

```diff
@@ -0,0 +1,311 @@
+          str7: {^RandomString: {length: 16}}
+          str8: {^RandomString: {length: 16}}
+          str9: {^RandomString: {length: 16}}
+          str10: {^RandomString: {length: 16}}
+          str11: {^RandomString: {length: 16}}
+          str12: {^RandomString: {length: 16}}
+          str13: {^RandomString: {length: 16}}
+          str14: {^RandomString: {length: 16}}
+          str15: {^RandomString: {length: 16}}
+          str16: {^RandomString: {length: 16}}
+          str17: {^RandomString: {length: 16}}
+          str18: {^RandomString: {length: 16}}
+          str19: {^RandomString: {length: 16}}
+          str20: {^RandomString: {length: 16}}
+          arr1: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr2: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr3: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr4: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr5: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr6: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr7: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr8: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr9: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr10: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr11: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr12: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr13: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr14: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr15: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr16: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr17: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr18: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr19: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          arr20: [{^Repeat: {count: 10, fromGenerator: {b: 0, c: 0}}}]
+          false1: false
+          false2: false
+          false3: false
+          false4: false
+          false5: false
+          false6: false
+          false7: false
+          false8: false
+          false9: false
+          false10: false
+          false11: false
+          false12: false
+          false13: false
+          false14: false
+          false15: false
+          false16: false
+          false17: false
+          false18: false
+          false19: false
+          false20: false
+
+
+# Phase 2: Ensure all data is synced to disk.
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: 1
+
+# Phase 3-N: Run variadic aggregate expression benchmarks
+- ExpressionAdd10:
```

Comment:
[nit] prefer numbers to be written out (so, "Ten" instead of "10")

Reply:
Done.